vscode-ext/src/extension.ts

```diff
@@ -139,6 +106,22 @@ async function checkVersion(version: string) {
 	});
 }
 
+async function parseNamespace() {
+	let filePath: string = await configFilePath();
+	let parsed;
+	if (filePath) {
+		const file = await (await vscode.workspace.fs.readFile(vscode.Uri.parse(filePath))).toString();
+		if (filePath.endsWith('json')) {
+			parsed = JSON.parse(file);
+		} else if (filePath.endsWith('yaml') || filePath.endsWith('yml')) {
```

Comment:
is it possible to assign the return value of an if statement to a variable in typescript?

Reply:
Only with ternary operators